src/lib/tzcode/CMakeLists.txt

```diff
@@ -1,2 +1,57 @@
-add_library(tzcode STATIC strftime.c strptime.c timelocal.c timezone.c)
+add_library(tzcode STATIC
+            strftime.c strptime.c timelocal.c localtime.c timezone.c)
 target_link_libraries(tzcode)
+
+set(TZCODE_PROJECT tzcode-iana)
+set(TZCODE_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR})
+set(TZCODE_PREFIX_DIR tz)
+set(TZCODE_PREFIX ${TZCODE_BUILD_DIR}/${TZCODE_PREFIX_DIR})
+set(TZDATA_TARGETS main.zi)
+set(TZCODE_SRC ${PROJECT_SOURCE_DIR}/src/lib/tzcode)
+set(TZLUA_SRC ${PROJECT_SOURCE_DIR}/src/lua)
+
+list(TRANSFORM TZDATA_TARGETS
```

Comment:
Discovered the harder way, that majority of our targets do not have `cmake` version which supports `list(PREPEND)` operation, so I've simplified it a bit (taking into account that there is single target we build here):

```diff
diff --git a/src/lib/tzcode/CMakeLists.txt b/src/lib/tzcode/CMakeLists.txt
index b43af125c..58bece584 100644
--- a/src/lib/tzcode/CMakeLists.txt
+++ b/src/lib/tzcode/CMakeLists.txt
@@ -6,13 +6,11 @@ set(TZCODE_PROJECT tzcode-iana)
 set(TZCODE_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR})
 set(TZCODE_PREFIX_DIR tz)
 set(TZCODE_PREFIX ${TZCODE_BUILD_DIR}/${TZCODE_PREFIX_DIR})
-set(TZDATA_TARGETS main.zi)
+set(TZDATA_TARGET main.zi)
 set(TZCODE_SRC ${PROJECT_SOURCE_DIR}/src/lib/tzcode)
 set(TZLUA_SRC ${PROJECT_SOURCE_DIR}/src/lua)
-
-list(TRANSFORM TZDATA_TARGETS
-     PREPEND ${TZCODE_PREFIX}/src/${TZCODE_PROJECT}/
-     OUTPUT_VARIABLE TZDATA_FULL_TARGETS)
+set(TZDATA_FULL_TARGET
+    ${TZCODE_PREFIX}/src/${TZCODE_PROJECT}/${TZDATA_TARGET})
 
 ExternalProject_Add(${TZCODE_PROJECT}
     URL https://data.iana.org/time-zones/releases/tzdb-2022a.tar.lz
@@ -29,11 +27,11 @@ ExternalProject_Add(${TZCODE_PROJECT}
         CC=${CMAKE_C_COMPILER}
         CFLAGS=${CMAKE_C_FLAGS}
         PACKRATDATA=backzone
-        make -C <SOURCE_DIR> ${TZDATA_TARGETS}
+        make -C <SOURCE_DIR> ${TZDATA_TARGET}
     BUILD_IN_SOURCE TRUE
     TEST_COMMAND ""
     INSTALL_COMMAND make DESTDIR=<INSTALL_DIR> install
-    BUILD_BYPRODUCTS ${TZDATA_FULL_TARGETS}
+    BUILD_BYPRODUCTS ${TZDATA_FULL_TARGET}
 )
 
 find_program(PERL perl)
@@ -42,7 +40,7 @@ add_custom_target(gen-timezones DEPENDS ${TZCODE_PROJECT})
 add_custom_command(TARGET gen-timezones
     COMMAND
     ${PERL} ${TZCODE_SRC}/gen-zone-abbrevs.pl
-        ${TZCODE_SRC}/zone-abbrevs.txt ${TZDATA_FULL_TARGETS}
+        ${TZCODE_SRC}/zone-abbrevs.txt ${TZDATA_FULL_TARGET}
         ${TZCODE_SRC}/timezones.h ${TZLUA_SRC}/timezones.lua
     COMMENT "Generating timezones.h and timezones.lua"
 )
```